src/subsets/Subsets.jsx

```diff
@@ -0,0 +1,263 @@
+        value={
+          varOption === "1"
+            ? this.state.selectActionTaken
+            : this.state.selectRace
+        }
+      />
+    );
+  }
+
+  onStateChange(selectedOption) {
+    return this.setState({
+      selectState: selectedOption,
+      selectMsaMds: "",
+      selectActionTaken: "",
+      selectRace: ""
+    });
+  }
+  onMsaMdsChange(selectedOption) {
+    return this.setState({
+      selectMsaMds: selectedOption
+    });
+  }
+
+  handleActionTakenUpdate(selectedOption) {
+    return this.setState({
+      selectActionTaken: selectedOption.length?"":selectedOption
+    });
+  }
+
+  handleRaceUpdate(selectedOption) {
+    return this.setState({
+      selectRace: selectedOption.length?"":selectedOption
+    });
+  }
+
+  render() {
+    const {location}  = this.props;
+    const subsetYear = location?location.pathname.split("/")[2]:"NA";
+
+    return (
+      <div className="Subsets">
+        <div className="intro">
+          <Header type={1} headingText="Subsets of HMDA data">
+            <p className="lead">
+              Customize your analysis of HMDA data. Create subsets of data here,
+              using pre-selected filters that allow you to compare queries. For
+              questions/suggestions, contact hmdafeedback@cfpb.gov.
+            </p>
+          </Header>
+        </div>
+        <table>
+          <thead />
+          <tbody>
+            <tr>
+              <th width="50%">Select a State (or Nationwide): </th>
+              <th width="50%">Choose an available MSA/MD:</th>
+            </tr>
+            <tr>
+              <td width="50%">
+                <Select
+                  onChange={this.handleStateUpdate}
+                  placeholder="Select a state..."
+                  searchable={true}
+                  autoFocus
+                  openOnFocus
+                  simpleValue
+                  options={stateOptions}
+                />
+              </td>
+              <td className="DropDown" width="50%">
+                {" "}
+                {this.loadMsaSelect(subsetYear)}
+              </td>
+            </tr>
+            <tr>
+              <th width="50%">Variable 1:</th>
+              <th width="50%">Variable 2:</th>
+            </tr>
+            <tr>
+              <td className="DropDown" width="50%">
+                {" "}
+                {this.loadVarSelect("1", "FIRST")}
+              </td>
+              <td className="DropDown" width="50%">
+                {" "}
+                {this.loadVarSelect("2", "SECOND")}
+              </td>
+            </tr>
+            <tr />
+            <tr>
+            <td  className="actionsTakenBoxes" width="50%">{this.getCheckBoxRows(actionsTakenOptions,this.state.selectActionTaken)}</td>
+            <td  width="50%">{this.getCheckBoxRows(raceOptions,this.state.selectRace)}</td>
+            </tr>
+          </tbody>
+          <tfoot />
+        </table>
+
+        <div className="Buttons">
```

Comment:
I think the back button should be removed and the "Next" button should just be something like "Get subset" since I believe that data will just render directly on the same page.